storey/flow.py

```diff
@@ -559,11 +560,12 @@ async def _worker(self):
                     self._pending_by_key[event.key].pending = []
 
                     task = self._safe_process_events(self._pending_by_key[event.key].in_flight)
-                    await self._q.put((event, asyncio.get_running_loop().create_task(task)))
+                    job = (event, asyncio.get_running_loop().create_task(task))
```

Comment:
Well what you are doing here is just prioritizing a specific key over the others.
If all keys keep coming all the time, you end up only processing `key1` first,  then `key2` etc

Reply:
Well, a bit... The producer will still block on the queue if there are more keys than queue slots. So there's still back pressure. Do we care about the exact prioritization of keys when writing to a table?